.github/workflows/containers-and-az-pool.yaml

```diff
@@ -148,44 +148,38 @@ jobs:
-      # Another example:
-      # https://github.com/Azure/azure-batch-cli-extensions/blob/cfd9439f93f319113ad221d81625e8fc7b32239b/samples/hello-world/create-pool-and-job/pool.json
       - name: Writing out config file
         run: |
           cat <<EOF > pool-config-${{ github.sha }}.txt
-          {
-              "pool": {
-                  "type": "Microsoft.Batch/batchAccounts/pools",
-                  "apiVersion": "2022-06-01",
-                  "properties": {
-                      "id": "${{ env.POOL_ID }}",
-                      "vmSize": "${{ env.VM_SIZE }}",
-                      "deploymentConfiguration": {
-                          "virtualMachineConfiguration": {
-                              "imageReference": {
-                                  "publisher": "microsoft-azure-batch",
-                                  "offer": "${{ env.VM_IMAGE_TAG }}",
-                                  "sku": "20-04-lts",
-                                  "version": "latest"
-                              },
-                              "nodeAgentSkuId": "${{ env.NODE_AGENT_SKU_ID }}",
-                              "containerConfiguration": {
-                                  "type": "dockercompatible",
-                                  "containerImageNames": "${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:test-${{ env.TAG }}",
-                                  "containerRegistries": [
-                                      {
-                                          "registryServer": "${{ env.REGISTRY }}",
-                                          "userName": "cfaprdbatchcr",
-                                          "password": "${{ secrets.CFAPRDBATCHCR_REGISTRY_PASSWORD }}"
-                                      }
-                                  ]
-                              }
-                          }
-                      }
-                  }
-              }
-          }
+          [Authentication]
+          vault_url =
+          keyvault_url =
+          tenant_id =
+          application_id =
+          subscription_id =
+          user_assigned_identity =
+          client_id =
+          participant_id =
+          subnet_id =
+          resource_group =
+
+          [Storage]
+          storage_account_url =
+          storage_account_name =
+          user_assigned_identity =
+
+          [Container]
+          container_registry_url = "https://${{ env.REGISTRY }}"
+          container_registry_username = cfaprdbatchcr
+          container_registry_password = ${{ secrets.CFAPRDBATCHCR_REGISTRY_PASSWORD }}
+          container_registry_server = ${{ env.REGISTRY }}
+
+          [Batch]
+          pool_vm_size = ${{ env.VM_SIZE }}
+          pool_id = ${{ env.POOL_ID }}
+          batch_account_name =
```

Comment:
Hey @jkislin, here is the information that the nnh pipelines python script needs to work. I think I added all the info I could. Can you help me adding whatever is missing?

Reply:
Ping @jkislin. Also, @zsusswein may have this information somewhere to add it as a secret to the repo?

Reply:
@gvegayon  so sorry - I never got this notification (or it got buried)
I'll add to my to-do for tomorrow.

Reply:
This is addressed (partially) in my #95

Reply:
Resolving as obsolete. Please re-open if I'm missing something!